webcompat/static/css/development/components/threesteps.css

```diff
@@ -38,21 +38,6 @@
   transform-origin: center;
 }
 
-.wc-Browserstack {
-  width: 100%;
-  background-image: url("../../../img/browserstack_logo.svg");
```

Comment:
Can we delete the svg file as well @ https://github.com/webcompat/webcompat.com/blob/master/webcompat/static/img/browserstack_logo.svg?

Reply:
uuh, forgot about that.